front/src/components/boxs/device-in-room/device-features/ThermostatDeviceFeature.jsx

```diff
@@ -0,0 +1,62 @@
+import get from 'get-value';
+import { Text } from 'preact-i18n';
+
+import { getDeviceName } from './utils';
+import { DeviceFeatureCategoriesIcon } from '../../../../utils/consts';
+
+const NumberDeviceFeature = ({ children, ...props }) => {
```

Comment:
Bad name according to filename.

An another option would be to merger with NumberDeviceFeature component, adding 2 options (step as number and displayUnit = true/false)

Reply:
I think we want to keep this as a separate component, as I changed the code to make it more specific :)